resource_estimator/src/estimates/data/job_params.rs

```diff
@@ -190,6 +190,6 @@ pub enum EstimateType {
 
 impl Default for EstimateType {
     fn default() -> Self {
-        Self::SinglePoint
+        Self::Frontier
```

Comment:
@msoeken , do you agree it is a good time to make it the default option?

Reply:
Decided to return the API back to SinglePoint by default but use the Frontier as the only option in the VS Code scenaition